pkg/util/util.go

```diff
@@ -188,3 +196,137 @@ func reportCancel(status reporter.CliStepStatus) {
 	})
 }
 
+func TestNetwork(ctx context.Context, kubeFactory kube.Factory, urls string) error {
```

Comment:
This is not the best api to pass multiple `urls` nobody needs to know that you end up concatenating them with `,`. You should use `, urls ...string)` here.